sample_data/france_by_metropolitan_region_since_2016/france_by_metropolitan_region_since_2016.md

```diff
@@ -0,0 +1,12 @@
+# Sources
+
+## france_by_metropolitan_region_since_2016.geojson
+Runfola D, Anderson A, Baier H, Crittenden M, Dowker E, Fuhrig S, et al. (2020) 
+geoBoundaries: A global database of political administrative boundaries. 
+PLoS ONE 15(4): e0231866. https://doi.org/10.1371/journal.pone.0231866. 
+Downloaded from: https://github.com/wmgeolab/geoBoundaries/tree/main/releaseData/gbOpen/FRA/ADM1 on 4 July 2022.
+
+## france_population_2008.csv
+Source: The National Institute of Statistics and Economic Studies (France). 2008 Population census. Downloaded 1 July 2022 from https://www.ined.fr/en/everything_about_population/data/france/population-structure/regions_departments/.
```

Comment:
There seem to be updated population data from 2022; we should use that.

Apart from that, the 2008 data seems to be different in the CSV. Bretagne 2018 population on the reference link is 2,802 ,519, whereas in the CSV it is 3,293,092.

Reply:
The updated population data is an estimate: as far as possible, I've attempted to use the latest official census data. I'll amend this if you prefer more recent data!